Library/Formula/libcutl.rb

```diff
@@ -0,0 +1,24 @@
+class Libcutl < Formula
+  homepage "http://www.codesynthesis.com/projects/libcutl/"
+  url "http://www.codesynthesis.com/download/libcutl/1.9/libcutl-1.9.0.tar.gz"
+  sha256 "1b575aa8ed74aa36adc0f755ae9859c6e48166a60779a5564dd21b8cb05afb7d"
+
+  depends_on "gcc" => :build
+
+  def install
+    system "./configure", "--disable-dependency-tracking",
+                          "--prefix=#{prefix}"
+    system "make", "install"
+  end
+
+  fails_with :clang do
+    cause <<-EOS.undent
+    Oh! It seems that you only have clang available, or GCC wasn't found!
+    Make sure you GCC is installed and recognized by homebrew.
+    EOS
+  end
+
+  test do
+    system "false"
```

Comment:
> Is there some specific reasoning behind you being the sole intended submitter?
Not at all. I'm not claiming being the intended submitter. Actually I'm on it since few months (August 2014) and we talked a long about how to compile, specific issues and else, by mailing-list with the author of the project. Then I was allowed to submit and maintain the packages and to use direct links to their files. Maybe the other dev was also allowed to submit, but that's not my point.

I wrote about me being allowed to maintain because I'm using direct links (and asked for permission, in case of it was not allowed).

Concerning the tests, I get it. But as I don't "make" the archives, am I allowed to write the test files (with `File.open`)? Or maybe download them as "patch" ?